skrl/agents/torch/ppo/ppo.py

```diff
@@ -388,55 +398,62 @@ def compute_gae(rewards: torch.Tensor,
             # mini-batches loop
             for sampled_states, sampled_actions, sampled_log_prob, sampled_values, sampled_returns, sampled_advantages in sampled_batches:
 
-                sampled_states = self._state_preprocessor(sampled_states, train=not epoch)
+                with torch.autocast(device_type=self._device_type, enabled=self._mixed_precision):
```

Comment:
Is it necessary to apply `autocast` to https://github.com/Toni-SM/skrl/blob/c15f3ce6f6d614d2e433439f9ed0606826a5e45a/skrl/agents/torch/ppo/ppo.py#L369-L373?

Reply:
Autocasting automatically chooses the precision for GPU operations to improve performance while maintaining accuracy [doc](https://pytorch.org/docs/stable/notes/amp_examples.html) 

removing the autocast will decrease the performance because float16 will not be used where it can be used.
I ran a small experiment:
| Scenario                                  | Time  |
|-------------------------------------------|-------|
| Autocast block as it is                   | 1:46  |
| No autocast                               | 2:10  |
| Autocast block only from "# compute value loss" | 1:50  |
| Autocast block only from "# compute entropy loss" | 1:49  |